soepy/test/test_unit.py

```diff
@@ -1,7 +1,9 @@
+import subprocess
 from collections import namedtuple
 
 import numpy as np
 
+
```

Comment:
Why the extra blank line?

Reply:
I ran black on the file so I am a little bit confused that there is an additional blank line myself. I will fix this in the next commit.

Reply:
Done.